From 41f2afe3485f2abd21ed1f9e18bdd4b782f38d73 Mon Sep 17 00:00:00 2001
From: David Li <li.davidm96@gmail.com>
Date: Fri, 9 Jan 2026 20:35:46 +0900
Subject: [PATCH] relax go requirement slightly

---
 go/adbc/go.mod | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/go/adbc/go.mod b/go/adbc/go.mod
index d95339b9d..6a44d724a 100644
--- a/go/adbc/go.mod
+++ b/go/adbc/go.mod
@@ -17,9 +17,7 @@
 
 module github.com/apache/arrow-adbc/go/adbc
 
-go 1.25.0
-
-toolchain go1.25.5
+go 1.24.11
 
 require (
 	cloud.google.com/go/bigquery v1.72.0
-- 
2.48.1

